src/aap_eda/tasks/ruleset.py

```diff
@@ -71,7 +74,10 @@ def activate(activation_id: int, requester: str = "User") -> None:
 
 @job("default")
 def deactivate(
-    activation_id: int, requester: str = "User", delete: bool = False
+    activation_id: int,
+    requester: str = "User",
+    delete: bool = False,
+    final_status: ActivationStatus = ActivationStatus.STOPPED,
```

Comment:
we can use `requester` to set final_status inside the function and pass it to `_perform_deactivate`, no need to pass through from outside, right?